lib/Conversion/TorchOnnxToTorch/DefaultDomainQtoZ.cpp

```diff
@@ -1952,20 +1996,6 @@ void mlir::torch::onnx_c::populateDefaultDomainQtoZ(
           }
         }
 
-        // Handle negative axis:
-        Value rankVal = rewriter.create<Torch::AtenDimOp>(binder.getLoc(),
-                                                          torchIntTy, data);
-        for (Value &axes : axesList) {
```

Comment:
You are removing support for negative dimensions specified in the axes list. This is wrong as we cannot guarantee when extracting it falls in the dimensions range.

Reply:
The negative dim will be handled in torch-to-linalg.